modules/machineset-creating-efa-options.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-creating-efa-options_{context}"]
+= Creating machines that use an AWS Elastic Fiber Adapter
+
+You can deploy compute machines that use an AWS Elastic Fiber Adapter (EFA) by adding the `networkInterfaceType` field to the machine set YAML file for your compute machines.
+
+.Prerequisites
+
+* You have configured security groups to support the use of an EFA.
++
+Machines that use an EFA must belong to security groups that allow all traffic between all hosts in the security group. Clusters that use the Machine API are not configured to meet this requirement by default.
++
+You can manually configure your security groups to support the use of an EFA by using the AWS Management Console or the AWS CLI. For more information, see the Amazon EC2 documentation about https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#updating-security-group-rules[updating security group rules].
+
+.Procedure
+////
+//Per Huali and Zhaohua, only the steps to set `networkInterfaceType` are required. Therefore, I am not sure what to do with these steps:
```

Comment:
If we know this feature isn't useful without this supplement, then I think it makes sense to include it